src/ui/NoNameFacet/NoNameFacetHeader/NoNameFacetHeader.ts

```diff
@@ -0,0 +1,76 @@
+import { $$ } from '../../../utils/Dom';
+import { l } from '../../../strings/Strings';
+import { SVGIcons } from '../../../utils/SVGIcons';
+import { SVGDom } from '../../../utils/SVGDom';
+import { INoNameFacetOptions } from '../NoNameFacetOptions';
+import { NoNameFacetHeaderClear } from './NoNameFacetHeaderClear';
+import { NoNameFacetHeaderCollapseToggle } from './NoNameFacetHeaderCollapseToggle';
+import { NoNameFacetHeaderOperatorToggle } from './NoNameFacetHeaderOperatorToggle';
+
+export interface INoNameFacetHeaderOptions {
```

Comment:
Seems a bit of a useless interface for now.

Could we simply use the `INoNameFacetOptions` type in the constructor ?

Reply:
Im pretty sure well need to add to it in the near future, why not keep it there for now?